content/docs/languages/cpp/_index.md

```diff
@@ -7,6 +7,4 @@ These language-specific pages are available:
 - [Quick start](quickstart)
 - [Basics tutorial](basics)
 - [Asynchronous-API tutorial](async)
-- API reference:
-  - [gRPC C++](https://grpc.github.io/grpc/cpp)
-  - [gRPC core](https://grpc.github.io/grpc/core)
```

Comment:
@ejona86 @jtattermusch: FYI, I dropped the reference to the gRPC Core API reference. It was something I added not to long ago, but it didn't seem to fit well anymore.